app/models/concerns/oregon_digital/ability/work_show_ability.rb

```diff
@@ -12,7 +12,28 @@ def work_show_ability
           cannot(%i[show], ActiveFedora::Base, visibility: 'osu') unless current_user.role?(osu_roles)
           cannot(%i[show], ActiveFedora::Base, visibility: 'uo') unless current_user.role?(uo_roles)
 
-          can(%i[show], ActiveFedora::Base) if current_user.role?(manager_permission_roles)
+          can :show, ActiveFedora::Base do |record|
+            show_record?(record)
+          end
+          can :read, SolrDocument do |solr_doc|
+            read_doc?(solr_doc)
+          end
+        end
+
+        def show_record?(record)
+          if record.suppressed?
+            current_user.role?(admin_permission_roles) && in_depositors_collection?(record.to_solr['edit_access_person_ssim'])
```

Comment:
`record.to_solr['edit_access_person_ssim']` is equivalent to `record.edit_users`